examples/rstudio/compare.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+percent_diff=$(compare -metric MSE "$1" "$2" null: 2>&1 > /dev/null | awk '{print $1}')
```

Comment:
This needs an explanation. What is `compare`? Does it introduce new dependencies, which should probably be added to `configure`? Why `-metric MSE`? What is the selected threshold (is it really zero, or rounded)? (If it's really zero, maybe we want `-metric AE`.) What is `awk` pulling out? Is it really a percentage (docs suggest the range is 0–1)? What is `null:`? Why are we discarding stderr?

Reply:
I'll add a comment and add that to the readme. Compare is a tool that's part of the image magick suite. I install image magick inside the container then do the comparison in the container with the output images to avoid requiring image magick as a new dependency. Although, image magick does come standard in most distros. So, it probably wouldn't affect many users if we set it as a dependency and ran this outside of the container. What do you prefer?

I chose MSE pretty arbitrarily. AE does look like a better fit. `awk` is just pulling out the MSE value. Image magick returns `0 (0)` but we only want the first value.